acra-censor/acra-censor_configuration_provider.go

```diff
@@ -50,6 +52,15 @@ func (acraCensor *AcraCensor) LoadConfiguration(configuration []byte) error {
 		return err
 	}
 	acraCensor.ignoreParseError = censorConfiguration.IgnoreParseError
+	acraCensor.parseErrorsLogPath = censorConfiguration.ParseErrorsLog
+	if acraCensor.parseErrorsLogPath != "" {
+		openedFile, err := os.OpenFile(acraCensor.parseErrorsLogPath, os.O_APPEND|os.O_RDWR|os.O_CREATE, 0600)
+		if err != nil {
+			return err
+		}
+		defer openedFile.Close()
```

Comment:
if we doesn't use this file after creation we can close it here without defer. but better to store file descriptor as censor field to avoid opening every write operation and reduce count of  syscalls (open/close)

Reply:
When and where should I close it is such case when acraserver stops?

Reply:
just here) `openedFile.Close()` instead `defer openedFile.Close()` because this file doesn't used anywhere after creation. As I understand here we just create file and check that we can do this and close it before next write.
But if we will store opened file in censor then we don't need `.Close()` here even with `defer`